docs/guidelines.md

```diff
@@ -113,7 +113,30 @@ ngeo.exampleDirective = function(…) {
       // …
     },
     controllerAs: 'ctrl',
+    // …
+  });
+```
+
+But if you bind your directive to the controller and you declare your attribute
+as a two ways binding attrribute, you can use the `.` notation. See the example
```

Comment:
I haven't added the 
``
/**
  @type {ol.Map}
  @export
*/
``

part because it was not existing in the previous example. This part concern AngularJS, not Closure at all. But I can add it in both part and explain that.

And I use `m` because it was as is in the previous example but I'll rename it as `map` that will be clearer. 

>Also note that in the colorpicker code the properties for scope are not given as strings.

Thanks for that. (Even if that works, that's not (longer?) necessary)